### PR TITLE
Issue 38

### PR DESCRIPTION
**Pull request by monapasan**
Additionally, I changed the path file name back to the relative one.
Please review and test.

### CODE REVIEW

**Comment by tzimmermann** [comment]
> Some of the tests are failing.
> @frontendphil @tzimmermann Can you please confirm that the test for `addLocation` is correct. I am a bit confused since the previous version was generating only a path without a `base` folder(e.g. if the file is under `repoFolder/src/someFolder/someFile.js` the reference path was `someFolder/someFile.js`) however according to the test, the path should be the full path to the file(e.g. `/Users/oleg/repo/src/someFile.js`)

I *think* that the test simply passes the full path to the `i18n-extract` method, while pex-client calls it with the relative path  `i18n-extract \"src/**/*.js\"`

**Comment by monapasan** [comment]
@tzimmermann even though `pex-client` calls it with the relative path, there is a difference in the output between 1.4.0 and 2.0.0 versions. The newer version puts the whole path into `.po` files if I execute it from within `pex-client` 🤷‍♀️

**Comment by tzimmermann** [comment]
> @tzimmermann even though `pex-client` calls it with the relative path, there is a difference in the output between 1.4.0 and 2.0.0 versions. The newer version puts the whole path into `.po` files if I execute it from within `pex-client` 🤷‍♀️

Yes but that's because of the API change, right? (the change in `filename.opts` that came in in `2.0.0`).

**Comment by monapasan** [comment]
@tzimmermann But the test should not be affected by it. In the test, the [path comes from node](https://github.com/signavio/i18n/blob/master/test/specs/scripts/extract.spec.js#L6)

**Comment by tzimmermann** [comment]
> I guess we should agree on whether we want to have a relative or full path file in the reference comment. After discussing it with @tzimmermann we agreed that having a relative path makes more sense. Are you ok with that @frontendphil? If yes, I will change the test accordingly so we can merge it.

Phil's mantra is "Don't ask for permission", so I'd say we do it with the relative path.

@monapasan Could you add a test for the original bug, i.e. that writes multiple times to the same file and checks that it's not overwritten?

**Comment by monapasan** [comment]
@tzimmermann Done! Please check 🙏

File: test/specs/scripts/extract.spec.js [hunk 1]
```diff
@@ -193,4 +193,20 @@ describe('extract', () => {
       expect(messages).to.contain('msgid "I got extracted"')
     })
   })
+
+  describe('includes message from multiple files', () => {
+    const multipleFilesLocaiton = `${fixtureDir}/multipleFiles`
```
Comment (tzimmermann) [inline test/specs/scripts/extract.spec.js]:
```suggestion
    const multipleFilesLocation = `${fixtureDir}/multipleFiles`
```

Comment (tzimmermann) [reply test/specs/scripts/extract.spec.js]: Just one typo, test is looking good!

Comment (monapasan) [reply test/specs/scripts/extract.spec.js]: ooops

**Pull request merged**